<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addasset</class>
 <widget class="QDialog" name="addasset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>706</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>526</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>员工信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_maintain">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_usable">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>710</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phonenumber">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>142</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>142</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_age">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_edubkg">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_jobTitile">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_dept">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>480</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_salary">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>540</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>81</y>
     <width>91</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_num">
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_usable">
      <property name="text">
       <string>年龄：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_requir">
      <property name="text">
       <string>电话号码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>学历:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_job">
      <property name="text">
       <string>职称:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>部门:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_salary">
      <property name="text">
       <string>工资:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>150</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
